download.md

```diff
@@ -20,15 +19,15 @@ You can find basic instructions for
 
 ## Linux
 
-[Tupelo 0.0.8 - Linux](tutorials/download/tupelo-v0.0.8-linux-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
-[Tupelo 0.0.8 - Linux - Checksum](tutorials/download/tupelo-v0.0.8-linux-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
+[Tupelo 0.0.9 - Linux](tutorials/download/tupelo-v0.0.9-linux-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
+[Tupelo 0.0.9 - Linux - Checksum](tutorials/download/tupelo-v0.0.9-linux-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
 
 ***
 
 ## OSX
 
-[Tupelo 0.0.8 - OSX](tutorials/download/tupelo-v0.0.8-darwin-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
-[Tupelo 0.0.8 - OSX - Checksum](tutorials/download/tupelo-v0.0.8-darwin-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
+[Tupelo 0.0.9 - OSX](tutorials/download/tupelo-v0.0.9-darwin-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
+[Tupelo 0.0.9 - OSX - Checksum](tutorials/download/tupelo-v0.0.9-darwin-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
```

Comment:
Super pedant alert: It’s called “macOS” now, not “OS X”.

Reply:
Changed.  Happy to be up to date 👍